static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -210,8 +212,83 @@ function SelectProjectStep({
   );
 }
 
+function AndroidInstallSteps() {
+  return (
+    <Fragment>
+      <li>
+        <StepTitle>{t('Update your projects SDK version')}</StepTitle>
+        <p>
+          {t('Make sure your SDKs are upgraded to at least version 6.0.0 (sentry-java).')}
+        </p>
+      </li>
+      <li>
+        <StepTitle>{t('Setup Performance Monitoring')}</StepTitle>
+        {t(
+          `For Sentry to ingest profiles, we first require you to setup performance monitoring.`
+        )}{' '}
+        <ExternalLink
+          openInNewTab
+          href="https://docs.sentry.io/platforms/java/performance/"
+        >
+          {t('Lear more about performance monitoring.')}
+        </ExternalLink>
+      </li>
+      <li>
+        <StepTitle>
+          {t('Enable profiling in your app by configuring the SDKs like below:')}
+        </StepTitle>
+        <CodeContainer>
+          {`<application>
+  <meta-data android:name="io.sentry.dsn" android:value="..." />
+  <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
+  <meta-data android:name="io.sentry.traces.profiling.enable" android:value="true" />
+</application>`}
+        </CodeContainer>
+      </li>
+    </Fragment>
+  );
+}
+
+function IOSInstallSteps() {
+  return (
+    <Fragment>
+      <li>
+        <StepTitle>{t('Update your projects SDK version')}</StepTitle>
+        <p>
+          {t(
+            'Make sure your SDKs are upgraded to at least version 7.23.0 (sentry-cocoa).'
+          )}
+        </p>
+      </li>
+      <li>
+        <StepTitle>{t('Setup Performance Monitoring')}</StepTitle>
+        {t(
+          `For Sentry to ingest profiles, we first require you to setup performance monitoring.`
+        )}{' '}
+        <ExternalLink openInNewTab href="https://docs.sentry.io/product/performance/">
```

Comment:
use the apple docs? https://docs.sentry.io/platforms/apple/performance/ or even the ios specific docs https://docs.sentry.io/platforms/apple/guides/ios/performance/